junit-reporting-handlebars/src/test/java/com/github/bogdanlivadariu/reporting/junit/builder/AllJunitReportsTest.java

```diff
@@ -14,7 +14,7 @@
 public class AllJunitReportsTest {
     List<String> xmlReports;
 
-    String reportPath = this.getClass().getClassLoader().getResource("valid-report-1.xml").getPath();
+    String reportPath = this.getClass().getClassLoader().getResource("valid-report-2.xml").getPath();
```

Comment:
I think it would be good to leave this test with valid-report-1 that does not relate to `<testsuites>` and add some extra checks for this case alone